README.md

```diff
@@ -10,56 +10,23 @@
 [![docs](https://img.shields.io/badge/docs-fuels.ts-brightgreen.svg?style=flat)](https://fuellabs.github.io/fuels-ts/)
 [![discord](https://img.shields.io/badge/chat%20on-discord-orange?&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2)](https://discord.gg/xfpK4Pe)
 
-## Table of contents
```

Comment:
Why remove the table of contents?

Reply:
Good question.

The contents of this `Readme` have shrunk considerably after several documentation iterations as more and more things were moved to the docs. Additionally, the Table of Contents occupied too much vertical space in the premium area of the Readme, which we can use to highlight immediate helpful information, IMO.

Following this principle, I reprioritized the content blocks inside the `Readme`, and moved what seemed more relevant to the top, starting with `Documentation`. I believe this is a _clean_ exit connecting the Repo and the Docs, so we don't need to repeat ourselves.

The Documentation site provides much better usability and readability.

<details>
  <summary>[see visual diff]</summary>

> **Note** _Later, we should fully decouple the repo `Readme` from the Docs `Front Page`._ (cc @Torres-ssf)


![diff](https://user-images.githubusercontent.com/26660/225739204-f99e73c3-404c-40ba-ac70-e52a48c2f04c.png)


</details>

Reply:
I think we lose on discoverability as the user needs to go to the documentation to find the things, but there are not hard points here.

Reply:
I see; it makes sense. I rolled it back to a middle ground: https://github.com/FuelLabs/fuels-ts/commit/edc0dfdfed6add5ab3b1c0f88e124ecd2b5175ab